.github/workflows/changelog.yml

```diff
@@ -18,26 +19,27 @@ jobs:
         uses: release-drafter/release-drafter@v5.15.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      # Generates a YAML changelog file using https://github.com/jenkinsci/jenkins-core-changelog-generator
-      - name: Generate YAML changelog draft
-        id: jenkins-core-changelog-generator
-        uses: jenkinsci/jenkins-core-changelog-generator@master
-        env:
-          GITHUB_AUTH: github-actions:${{ secrets.GITHUB_TOKEN }}
-      - name: Upload Changelog YAML
-        uses: actions/upload-artifact@v2.2.4
+  jenkins_io_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
         with:
-          name: changelog.yaml
-          path: changelog.yaml
-#TODO(oleg-nenashev): It will not work well with Release Drafter which does not recreated releases. Asset does not get overwritten, and there is no ready-to-go API for overriding assets
-      # Upload YAML to the release draft assets
-#      - name: Upload changelog.yaml to the Release Draft
-#        id: upload-changelog-yaml
-#        uses: actions/upload-release-asset@v1.0.1
-#        env:
-#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-#        with:
-#          upload_url: ${{ steps.release-drafter.outputs.upload_url }}
-#          asset_path: ./changelog.yaml
-#          asset_name: changelog.yaml
-#          asset_content_type: text/yaml
+          app_id: ${{ secrets.JENKINS_DEPENDENCY_UPDATER_APP_ID }}
+          private_key: ${{ secrets.JENKINS_DEPENDENCY_UPDATER_PRIVATE_KEY }}
+          repository: timja/jenkins.io
+      - name: Check out
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Publish jenkins.io changelog draft
+        env:
+          GITHUB_TOKEN: ${{ steps.generate-token.outputs.token }}
+          GITHUB_AUTH: "nobody:${{ steps.generate-token.outputs.token }}"
+          GIT_AUTHOR_NAME: jenkins-dependency-updater
+          GIT_AUTHOR_EMAIL: <81680575+jenkins-dependency-updater[bot]@users.noreply.github.com>
+          GIT_COMMITTER_NAME: jenkins-dependency-updater
+          GIT_COMMITTER_EMAIL: <81680575+jenkins-dependency-updater[bot]@users.noreply.github.com>
+        run: |
+          wget --quiet https://raw.githubusercontent.com/timja/core-changelog-generator/automate-changelog-generation/generate-weekly-changelog.sh
```

Comment:
branch and repo need updating after merge of https://github.com/jenkinsci/core-changelog-generator/pull/14